scripts/prepare.sh

```diff
@@ -27,6 +27,16 @@ mkdir -p android/app/build/intermediates/merged_assets/debug/out/saplingtree/mai
 cp -R node_modules/edge-currency-accountbased/lib/zcash/zecCheckpoints/ android/app/build/intermediates/merged_assets/debug/out/saplingtree/mainnet 2>/dev/null || :
 cp -R node_modules/edge-currency-accountbased/lib/zcash/zecCheckpoints/ ios/Pods/ZcashLightClientKit/Sources/ZcashLightClientKit/Resources/saplingtree-checkpoints/mainnet 2>/dev/null || :
 
+# Copy Firebase configs
+if [ ! -f "GoogleService-Info.plist" ]; then
+  cp GoogleService-Info.sample.plist GoogleService-Info.plist
+fi
+cp GoogleService-Info.plist ios/edge/GoogleService-Info.plist
+if [ ! -f "google-services.json" ]; then
+  cp google-services.sample.json google-services.json
+fi
+cp google-services.json android/app/google-services.json
+
```

Comment:
Since `GoogleService-Info.plist` is for iOS and `google-services.json` is for Android, maybe we should leave them in the original locations, and avoid the double-copy.

So we merely rename the sample files to `ios/edge/GoogleService-Info.sample.plist` and `android/app/google-services.sample.json`, and then copy the samples on top of the real files with the `if [ ! -f ...]` guard. That keeps our root directory clean, and will work with the current Jenkins setup as well.